assets/js/blocks/product-query/inspector-controls/upgrade-notice.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Notice, Button } from '@wordpress/components';
+
+const FormattedNotice = ( { notice }: { notice: string } ) => {
+	const strongText = 'Product Collection';
```

Comment:
Needs translation:
```suggestion
	const strongText = __( 'Product Collection', 'woo-gutenberg-products-block' );
```

Reply:
Good catch! The idea was that it would be replaced on the text that's been already translated, but that means "Product Collection" could've been a different text as well and it wouldn't work. 

Changed that logic in this [commit](https://github.com/woocommerce/woocommerce-blocks/pull/10268/commits/0058528c964296b3f811fc48654b028f4be3b0aa).